.github/workflows/assign-tech-writers.yml

```diff
@@ -0,0 +1,15 @@
+name: Assign tech writers
+
+on: 
+  pull_request: 
+    types: [labeled]
+jobs:
+  assign_tech_writers:
+    if: ${{github.event.label.name == 'doc-changes'}}
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: assign techwriters to PR
+        uses: kentaro-m/auto-assign-action@v1.2.0
+        with:
+          configuration-path: ".github/auto_assign.yml"
```

Comment:
Do we need a task to remove tech writers if the label is removed or will we just remember to do that? We can probably remember, it's probably not very common.

Reply:
It's nice to have, but is probably rarely useful. So it's probably better to keep the workflow simple and not support removing people.